docker-compose.yml

```diff
@@ -2,43 +2,32 @@ version: "3"
 
 services:
   db:
```

Comment:
Maybe it's a good idea to expose `5432:5432`

Or fw it to `15432` (to avoid errors if host has already a running postgres at 5432)

What do you think?

Reply:
Done